app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class OmniauthCallbacksController < ApplicationController
+  def login_dot_gov
+    @user = User.from_omniauth(request.env['omniauth.auth'])
+    return unless @user.persisted?
```

Comment:
There should be a spec for the a scenario where the `@user` record is not persisted for whatever reason.

Reply:
Adding this to SRCH-917. If user is not persisted they are not authorized that method will need to redirect to an error page and rspec tests will cover that.

Reply:
But I did add a basic check for now. It will get rewritten when I work on SRCH-917.